sandy/core/records.py

```diff
@@ -327,9 +327,40 @@ def write_int(x):
 
 
 def line_numbers(length, istart=1):
+    """
+    Line number creator
+
+    Parameters
+    ----------
+    length : 'int'
+        Number of lines.
+    istart : 'int', optional
+        Number from where start counting. The default is 1.
+
+    Returns
+    -------
+    ilines : `list`
+        List containing the number of each line.
+
+    Examples
+    --------
+    >>> np.array(line_numbers(1.0e6)).max()
+    99999
+    >>> np.array(line_numbers(1.0e6+1)).min()
+    1
+    >>> np.array(line_numbers(1.0e4+1)).max()
+    10001.0
+    """
     iend = istart + length
-    ilines = np.arange(istart, iend)
-    return np.where(ilines < 100000, ilines, ilines-99999).tolist()
+    if length < 100000:
```

Comment:
I would do something much simpler, like
`np.tile(np.arange(1, 1e5, dtype=int), int(iend//99999)+1)[:iend]`

I am not even sure we need `istart`